examples/plotting/file/line_toggle.py

```diff
@@ -0,0 +1,40 @@
+"""
+Example demonstrating the picking of line objects. The transparency of
+the lines is toggled when clicked.
+"""
+
+import numpy as np
+from bokeh.plotting import output_file, show, figure, Figure
+from bokeh.models import TapTool, CustomJS
+
+# Normally, the use of the tap tool causes Bokehjs to "select" a line
+# when it is clicked on, which is not what we want in this example. To
+# deal with this, we modify the "selected" object to make it look as
+# if no glyph was clicked.
```

Comment:
I don't understand this comment either, you're modifying the glyph renderer, not the selection object.